core/templates/dev/head/services/StateTopAnswersStatsServiceSpec.ts

```diff
@@ -34,6 +42,27 @@ require('App.ts');
 require('pages/exploration-editor-page/services/exploration-states.service.ts');
 require('services/StateTopAnswersStatsService.ts');
 
+class MockAnswerStats {
+  answer: any;
+  answerHtml: string;
+  frequency: number;
+  isAddressed: boolean;
+  constructor(
+      answer: any, answerHtml: string, frequency: number,
+      isAddressed: boolean) {
+    this.answer = angular.copy(answer);
+    this.answerHtml = answerHtml;
+    this.frequency = frequency;
+    this.isAddressed = isAddressed;
+  }
+  toBackendDict() {
```

Comment:
What’s the return type here? Also, make an interface for this return type.

Reply:
Sorry, I had the wrong assumption here, I take that comment back.

Reply:
Because the service above is still in AngularJS, declare with an anonymous type instead (so what you did in the comment above).

Reply:
Done!